examples/experimental/dagster-airlift/examples/tutorial-example/README.md

```diff
@@ -464,6 +463,214 @@ export AIRFLOW_HOME="$TUTORIAL_EXAMPLE_DIR/.airflow_home"
 dagster dev -f migrate.py
 ```
 
+## Adding asset checks
+
+Once you have peered your Airflow DAGs in Dagster, regardless of migration progress, you can begin to add asset checks to your Dagster code. Asset checks can be used to validate the quality of your data assets, and can provide additional observability and value on top of your Airflow DAG even before migration starts.
```

Comment:
Let's lead with the DAG acceptance tests as that example is simpler and more minimal.